configure.ac

```diff
@@ -133,7 +134,6 @@ AS_CASE([$host],
 
     # externals are dynamically linked to pd.dll in their individual automake files
     EXTERNAL_CFLAGS="-mms-bitfields"
-    EXTERNAL_LDFLAGS="-s -Wl,--enable-auto-import -no-undefined -lpd"
```

Comment:
I would keep the platform-specific link flags here. Can't we just remove the `-s` all together? I think the question of stripping should be confined to `make app`.

Reply:
If I understand correctly what you suggest is removing the EXTERNAL_LDFLAGS "-s" completly.
I really have no idea of the effect since I was much more focus on core than on external in the development for now.
What I can propose is to :
1) Leave the EXTERNAL_LDFLAGS unchanged and remove all that modify EXTERNAL_LDFLAGS from my PR
2) remove the -s only from this line and prepend it in the AS_IF debug section later if debug is yes.

Reply:
> If I understand correctly what you suggest is removing the EXTERNAL_LDFLAGS "-s" completly

Yes. There is no reason why we should strip the binaries by default. Stripping only makes sense if you want to distribute the binaries and save some space, and for that you would need `make app` anyway. Hence my suggestion that stripping should only be done by `make app` (if requested).

> remove the -s only from this line and prepend it in the AS_IF debug section later if debug is yes.

`-s` means "strip". You certainly don't want to do that for a debug build :-)

Reply:
(Actually, I don't even understand how the `-s` got there in the first place... After all, the Pd core is *not* stripped by default.)

Reply:
Ok will remove the -s then from EXTERNAL_LDFLAGS

Reply:
Should be ok now but strangely the review is not mark Outdated...
Could you please have a check and resolve this one after ?

Reply:
Yeah, that's weird.